src/components/UserIcon/UserIcon.module.css

```diff
@@ -11,5 +11,20 @@
   color: #ffffff;
   text-align: center;
   text-transform: uppercase;
-  margin-right: 1rem;
+}
+
+.lg {
+  --userIcon-size: 2.5rem;
```

Comment:
Nice!. smart use of css-variables. 

Not important, but is the "userIcon-size" on line 2 necessary? 
if it needs to be there, the default size is "md", so maybe it should be 2rem by default here as well?  

`.userIcon {
  --userIcon-size: 2.5rem;
  min-width: var(--userIcon-size);`

Reply:
I'm not sure if it's needed actually... 🤔
I'm just used to variable always having to be declared before they're used 😅

But you're absolutely right! It should at least be set to the default value.
I'll fix it tomorrow 😊